helix-term/src/application.rs

```diff
@@ -228,6 +229,63 @@ impl Application {
                     Notification::LogMessage(params) => {
                         log::warn!("unhandled window/logMessage: {:?}", params);
                     }
+                    Notification::ProgressMessage(params) => {
```

Comment:
Wouldn't this hides some other log error messages?

Reply:
Do you mean the `Notification::LogMessage` branch?

Reply:
Yeah

Reply:
There's some very low level log messages submitted in `LogMessage` (at least with gopls), I think we should catch them and display them with https://github.com/helix-editor/helix/issues/149 later on.

Reply:
Or maybe we can hold them. Either way, let's merge this for now.